requirements-minimal.txt

```diff
@@ -1,18 +1,17 @@
 pynwb>=1.4.0
 hdmf>=3.2.1
+h5py>=2.10.0
 tqdm>=4.60.0
 natsort>=7.1.1
 numpy>=1.21.0
-pandas>=1.2.3
-scipy>=1.4.1
-h5py>=2.10.0
 PyYAML>=5.4
 jsonschema>=3.2.0
 psutil>=5.8.0
-lxml>=4.6.5
+dandi>=0.34.1
+nwbinspector>=0.2.3
 spikeextractors>=0.9.9
 spikeinterface>=0.93.0
 neo>=0.9.0
 roiextractors>=0.4.0
-dandi>=0.34.1
-nwbinspector>=0.2.3
+lxml>=4.6.5
+scipy>=1.4.1
```

Comment:
Eventually, goal is to have these removed as well for super minimal install.

Justification being, if someone just wants to use nwbct for ophys, why require them to install spikeinterface and neo as well?